### DISCUSSION TITLE
Using Renovate to update Docker Images from public.ecr.aws

### DISCUSSION THREAD

**Discussion by fcbear** [renovatebot/renovate#16038] [Request Help] [answered]
Im currently trying to get renovate to update the docker file.
We mainly use images hosted by public.ecr.aws for this case.
After a long time of looking around, all I found were ways using the non public one.

Anyone got a working solution for the host rules for public.ecr.aws - or are we forced to use private aws access?

**Comment by rarkins** [comment]
IIRC the public ECR doesn't include tags listing, which means Renovate can't look up tags/versions.

**Reply by morremeyer** [reply]
This was requested in https://github.com/aws/containers-roadmap/issues/1262 and released by AWS on 2022-07-22, see https://github.com/aws/containers-roadmap/issues/1262#issuecomment-1196017565. 

So it should now work.

@fcbear, if you have a test case for this still, would you care giving us an update after testing?

**Reply by viceice** [reply]
@morremeyer please create a new answer, so we can mark it as new now correct answer.

**Reply by morremeyer** [reply]
@viceice done!

**Comment by morremeyer** [comment, marked as answer]
This was requested in https://github.com/aws/containers-roadmap/issues/1262 and released by AWS on 2022-07-22, see https://github.com/aws/containers-roadmap/issues/1262#issuecomment-1196017565. 

So it should now work.

@fcbear, if you have a test case for this still, would you care giving us an update after testing?
